gordo_components/__init__.py

```diff
@@ -51,12 +51,3 @@ def _parse_version(version: str) -> Tuple[int, ...]:
 except Exception:
     warnings.warn(f"Failed to fix absl logging bug {traceback.format_exc()}")
     pass
-
-
-# Set log level, defaulting to INFO
-log_level = os.getenv("LOG_LEVEL", "INFO").upper()
-
-logging.basicConfig(
-    level=getattr(logging, log_level),
-    format="[%(asctime)s] %(levelname)s [%(name)s.%(funcName)s:%(lineno)d] %(message)s",
-)
```

Comment:
Were these going into the cli on this PR as per #773?

Reply:
Yep yep, just haven't rebased as it was addressed in #774